### PR TITLE
Updated filter response regexes to support Mesos/Marathon upgrade

### PR DESCRIPTION
**Pull request by amitbose327**

### CODE REVIEW

File: aaad/marathon.py [hunk 1]
```diff
@@ -17,13 +17,13 @@ def getName(self):
...
     def filterResponseBody(self, body, allowed_namespaces, request_uri):
-        uri_pattern = re.compile("^{}$".format("/marathon/*v2/apps"))
+        uri_pattern = re.compile("^{}$".format("/marathon/*v2/apps.*"))
         uri_match = uri_pattern.match(request_uri)
         if uri_match:
             response = self.filterV2AppsResponse(body, allowed_namespaces)
             return response
 
-        uri_pattern = re.compile("^{}$".format("/marathon/*v2/groups"))
+        uri_pattern = re.compile("^{}$".format("/marathon/*v2/groups.*"))
```
Comment (jhulten) [inline aaad/marathon.py]: What is the value of the format in this case?

File: aaad/marathon.py [hunk 2]
```diff
@@ -17,13 +17,13 @@ def getName(self):
         return "Marathon"
 
     def filterResponseBody(self, body, allowed_namespaces, request_uri):
-        uri_pattern = re.compile("^{}$".format("/marathon/*v2/apps"))
+        uri_pattern = re.compile("^{}$".format("/marathon/*v2/apps.*"))
```
Comment (jhulten) [inline aaad/marathon.py]: So `/marathon/*v2/apps.*` implies that the following are accepted:

- /marathon/v2/apps
- /marathonv2/apps
- /marathon/////v2/apps

Is that the intent?

Comment (amitbose327) [inline aaad/marathon.py]: Yes

**Comment by jhulten** [comment]
LGTM

**Pull request merged**